models/src/main/java/com/vimeo/networking2/enums/SlackUserPreference.kt

```diff
@@ -0,0 +1,27 @@
+package com.vimeo.networking2.enums
+
+/**
+ * The preference for which changes a user wants the Slack channel to be notified.
+ */
+enum class SlackUserPreference(override val value: String?) : StringValue {
```

Comment:
Add this to `EnumsTest`

Reply:
Good call out. I looked over the API docs and realized our `Folder` implementation was missing the slack related properties, so I added them.